share/spack/docker/README.md

```diff
@@ -0,0 +1,20 @@
+Using this docker container for development.
+------------
+
+    $ cd share/spack/docker
+    $ docker-compose run --rm ubuntu
+
+This command should drop you into an interactive shell where you can run spack
+within an isolated docker container running ubuntu.  The copy of spack being
+used should be tied to the working copy of your cloned git repo, so any changes
+you make should be immediately reflected in the running docker container.
+
+To work within a container running a different linux distro, change the "ubuntu"
+argument to any one of the services listed under the ``docker-compose.yml``
```

Comment:
I think the double "`" is rst not md :)

Reply:
if you write this file in full `.rst` instead of `.md` one can also easier include it into the main sphinx docs into the developer workflows :)